File: WeightCaptureApplication/app/src/main/java/com/ricken/weightcaptureapplication/database/object/Scale.java
package com.ricken.weightcaptureapplication.database.object;

import com.ricken.weightcaptureapplication.IdElement;

public class Scale extends IdElement {
    private String name;
    public Scale(){}
    public Scale(String name){
        this.name = name;
    }
    public String getName(){
        return name;
    }
    public void setName(String name){
        this.name = name;
    }
    public String toString(){
        return getName();
    }
}

File: WeightCaptureApplication/app/src/main/java/com/ricken/weightcaptureapplication/database/object/Scale.kt
package com.ricken.weightcaptureapplication.database.`object`

import com.ricken.weightcaptureapplication.IdElement

class Scale : IdElement {
    var name: String? = null

    constructor()
    constructor(name: String?) {
        this.name = name
    }

    override fun toString(): String {
        return name!!
    }
}